backend/src/components/permissionUtils.js

```diff
@@ -290,6 +315,26 @@ function findSdcDistrictCollectionID_params(req, res, next) {
   return next();
 }
 
+function findSInstituteTypeCollectionID_body(req, res, next) {
```

Comment:
Is there a downstream function after this that requires that the res.locals.requestedInstituteType values to be populated? If not, setting the values here can probably be removed.

Reply:
Yes, a downstream function is using thosde values